scene/gui/texture_progress_bar.cpp

```diff
@@ -608,10 +608,10 @@ int TextureProgressBar::get_fill_mode() {
 }
 
 void TextureProgressBar::set_radial_initial_angle(float p_angle) {
-	while (p_angle > 360) {
-		p_angle -= 360;
+	if (p_angle > 360) {
+		p_angle = Math::fmod(p_angle, 360);
```

Comment:
This returns `NaN` if `p_angle` id `+/-INF`

Reply:
Yes.
Any suggestions how it should be handled?

Reply:
Something like that?
```
if (!Math::is_finite(p_angle)){
	ERR_PRINT("radial_initial_angle should be finite number.");
}
```

Reply:
Like this:
```cpp
ERR_FAIL_COND_MSG(!Math::is_finite(p_angle), "Angle is non-finite.");
```